packages/blade/src/components/VisuallyHidden/_KitchenSink.VisuallyHidden.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { composeStories } from '@storybook/react';
```

Comment:
I believe we don't need a kitchen sink story for this as this component doesn't show up in UI.

Reply:
Rehne dete hain, what if suddenly <VisuallyHidden \/> becomes <OhShitItsNotHiddenAnymore \/>

Reply:
yes parity